backend/Techfesia2019/Techfesia2019/urls.py

```diff
@@ -24,6 +24,10 @@
     path('admin/', admin.site.urls),
     path('docs', schema_view),
     path('auth/', include("registration.urls")),
+    path('users/<str:username>/invitation/', include('event_registrations.invitation_urls')),
```

Comment:
change to `users/<username>/invitations`. Maybe we need to change docs as well

Reply:
Ok